package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "name-request",
-  "version": "5.3.9",
+  "version": "5.3.10",
```

Comment:
You update this file manually, didn't you? Don't do that.

Do this instead: `npm i`.

Reply:
If I used 'npm i', package-lock.json has lots of changes. Such as:
![image](https://github.com/bcgov/namerequest/assets/54647458/0a6bb4d4-8b5b-472e-9f41-f0bbecf45938)

Reply:
Regardless, this file is not currently updated correctly.

Reply:
OK, now it looks fine, but I'm concerned about "lots of changes" that you see. Please ensure that you are using npm v9.8.0.